### ISSUE TITLE
Provide an option to use different typescript instance

### ISSUE THREAD

**Issue by foisonocean** [wessberg/rollup-plugin-ts#45]
My use case is in a big mono repo, there are many projects in this repo and all projects share same build tool. Each project may depends on different typescript version. So I hope there could be an option like this:

```javascript
import ts from 'rollup-plugin-ts'
export default {
  // ...
  plugins: [
    ts({
      tsInstance: require("myProjectPath/node_modules/typescript"),
    }),
  ],
}
```

**Comment by wessberg** [comment]
Just wanted to give you a heads up that I'm working on this feature. You can expect it to be released with the next patch.

**Comment by wessberg** [comment]
This has been released as part of [v1.2.0](https://github.com/wessberg/rollup-plugin-ts/releases/tag/v1.2.0). It is documented [here](https://github.com/wessberg/rollup-plugin-ts#typescript). Happy New Year!
